lua/nvim-treesitter/textobjects/attach.lua

```diff
@@ -18,7 +18,13 @@ function M.make_attach(normal_mode_functions, submodule)
         end
         if query then
           local cmd = ":lua require'nvim-treesitter.textobjects."..submodule.."'."..function_call.."('"..query.."')<CR>"
-          api.nvim_buf_set_keymap(bufnr, "n", mapping, cmd, {silent = true, noremap = true })
+
+          if repeatable then
+            local fn = 'nvim_treesitter_textobjects#repeatable("'..cmd..'")'
+            api.nvim_buf_set_keymap(bufnr, "n", mapping, fn, {silent = true, noremap = true, expr = true })
+          else
+            api.nvim_buf_set_keymap(bufnr, "n", mapping, cmd, {silent = true, noremap = true })
+          end
```

Comment:
This looks very nice to do it for all possible functions! But it will not work when calling the Lua function directly or when using the command the command :TSTextobjectSwapNext. It's also a bit strange that you can only make all functions of a module repeatable (which would still work for our current use case.